script.kodi.android.update/default.py

```diff
@@ -142,9 +142,10 @@ def buildItems(self, url):
             for item in self.getItems(soup):
                 try: #folders
                     label, label2 = re.compile("(.*?)/-(.*)").match(item).groups()
-                    if label == PLATFORM: label2 = LANGUAGE(30014)%PLATFORM
+                    print('buildItems',label,label2)
```

Comment:
ups I missed this one, please remove in the next version

Reply:
Okay, thx